low-level/framework/base/sspl_constants.py

```diff
@@ -61,14 +61,18 @@
 # required only for init
 component = 'sspl/config'
 file_store_config_path = '/etc/sspl.conf'
+roles = ['vm', 'cortx', 'ssu', 'gw', 'cmu']
 salt_provisioner_pillar_sls = 'sspl'
 salt_uniq_attr_per_node = ['cluster_id']
 salt_uniq_passwd_per_node = ['RABBITMQINGRESSPROCESSOR', 'RABBITMQEGRESSPROCESSOR', 'LOGGINGPROCESSOR']
+# This file will be created when sspl is being configured for node replacement case
```

Comment:
remove commented out code, which you are introducing

Reply:
this is required for sppl_config anyway so kept it uncommented.